docs/src/components/ColorBox.tsx

```diff
@@ -14,6 +14,7 @@ import CopyToClipboard from './CopyToClipboard'
 type ColorBoxProps = {
   children?: React.ReactNode
   name?: string
+  header?: boolean
```

Comment:
the name of the prop is a bit misleading when this is used on the client side:

```jsx
  <ColorBox
        header={true}  // what is the effect of that? and it is tempting to write like 'header="my header"'
        name={token}
        key={`${token}schema`}
        size="small"
```

Maybe we could come up with some better option? What are the cases this prop should differentiate? According to implementation, it is about  rendering copy to clipboard icon or not. Maybe we should name it accordingly, like `copyToClipboardIcon={true}`?